app/frontend/src/features/map/mapActions.ts

```diff
@@ -0,0 +1,18 @@
+import { createAsyncThunk } from "@reduxjs/toolkit";
+import { service } from "../../service";
+
+export const onLand = createAsyncThunk(
```

Comment:
What do you forsee these being used for? At the moment they don't need to be in thunks, as they don't use the redux store. We are getting rid of redux anyway :P

Reply:
Ahahah, see I don't know. I don't understand this thunk stuff yet...

The functions themselves are not yet used, and I'm not sure if they will ever have a use directly from the frontend. (Only as checks e.g. to make sure you don't say you live, or make an event in the middle of the ocean, etc). It's just nice to click on the map and try out how these work...